influxql/select.go

```diff
@@ -850,13 +857,11 @@ func (b *exprIteratorBuilder) buildCallIterator(expr *Call) (Iterator, error) {
 			}
 			return itr, nil
 		case "median":
-			opt := b.opt
-			opt.Ordered = true
-			input, err := buildExprIterator(expr.Args[0].(*VarRef), b.ic, b.sources, opt, false)
+			input, err := buildOrderedExprIterator(expr.Args[0].(*VarRef), b.ic, b.sources, b.opt, false)
 			if err != nil {
 				return nil, err
 			}
-			return newMedianIterator(input, opt)
+			return newMedianIterator(input, b.opt)
```

Comment:
It looks like the reduce iterator does actually look at this variable. While I don't view it as essential (likely a no-op), I'm going to modify this back to its original. It's likely not worth the risk.